app/components/charts/item-location-sales-trend/component.js

```diff
@@ -1,39 +1,56 @@
-import Ember from 'ember';
-import computed from 'ember-computed-decorators';
+import Ember from "ember";
+import computed from "ember-computed-decorators";
 
 const {
   and,
   notEmpty
 } = Ember.computed;
 
 export default Ember.Component.extend({
-  classNames: ['col', 'card-1'],
-  classNameBindings: ['shouldDisplay::hidden'],
-  hasData:       notEmpty('salesData'),
-  hasItem:       notEmpty('item'),
+  localStorage: Ember.inject.service(),
+
+  classNames: ["col", "card-1"],
+  classNameBindings: ["shouldDisplay::hidden"],
+  hasData:       notEmpty("salesData"),
+  hasItem:       notEmpty("item"),
   shouldDisplay: and("hasData", "hasItem"),
+  page:"sales_orders",
+  isHidden:true,
+
+  async willRender(){
```

Comment:
Instead of needing this willRender, we should just bind directly to the pref. This could allow us to remove this whole willRender